<launch>

  <arg name="sensor1_type" default="lidar"/>
  <arg name="sensor2_type" default="mono"/>
  <arg name="sensor1_id" default="0"/>
  <arg name="sensor2_id" default="0"/>

  <node pkg="calibration" exec="velo2cam_calibration" name="velo2cam_calibration" output="screen">
    <!-- lidar_pattern, mono_qr_pattern, stereo_pattern -->
    <remap from="~cloud1" to="$(var sensor1_type)_pattern_$(var sensor1_id)/centers_cloud"/>
    <remap from="~cloud2" to="$(var sensor2_type)_pattern_$(var sensor2_id)/centers_cloud"/>
    <param name="is_sensor1_cam" value="false" if=1/>
    <param name="is_sensor2_cam" value="false" />
    <param name="is_sensor1_cam" value="true" />
    <param name="is_sensor2_cam" value="true" />
    <!-- <param name="is_sensor1_cam" value="false" unless="$(eval sensor1_type == 'mono' or sensor1_type == 'stereo')"/>
    <param name="is_sensor2_cam" value="false" unless="$(eval sensor2_type == 'mono' or sensor2_type == 'stereo')"/>
    <param name="is_sensor1_cam" value="true" if="$(eval sensor1_type == 'mono' or sensor1_type == 'stereo')"/>
    <param name="is_sensor2_cam" value="true" if="$(eval sensor2_type == 'mono' or sensor2_type == 'stereo')"/> -->
  </node>
</launch>
